docs/language.md

```diff
@@ -280,3 +282,61 @@ and dynamic maps.
 * `Object.keys()` is currently not implemented for classes; when it will be
   the order of fields will be static declaration order
 * how to validate types of C++ classes (Pin mostly)?
+
+
+# Python
+
+## Supported language features
+The following language features should be fully supported and work according to the Python 3 language specification.
+
+* lists
+* dictionaries
+* function definitions
+* function calling
+* method calling
+* calling into any MakeCode library code
+* literals: strings, numbers, boolean, None
+* many list methods: pop, clear, index, count, len
+* many string methods: casefold, capitalize, center, count, endswith, find, index, isalnum, isalph, isascii, isdigit, isnumeric, isspace, isdecimal, isidentifier, islower, isprintable, istitle, issupper, join, ljust, lower, lstrip, replace, rfind, rindex, rjust, rsplit, rstrip, split, splitlines, startswith, strip, swapcase, title, upper, zfill, ord
+* many math functions: int, min, max, abs, randint
+* while loop
+* for-in loop with range(), array or string literal
+* break, continue
+* conditional statements (if, elif, else)
+* pass statement
+* variables*
+* if expression / ternary operator
+* comparison (in, notin)
+* byte literal
+* *args
+* type annotations using ":" syntax
+* slice notation**
+* lambda
+
+*: variable semantics have slightly different scoping rules than Python 3 and global & nonlocal keywords are unsupported.
+**: some slice notation is not yet supported
+
+## Not supported language features
+The following language features are not yet supported.
+
+* with
+* assert
+* classes
+* __constructor
+* super()
+* global & nonlocal
+* raise
+* try
+* generators
+* attributes
+* import, import from
+* sets
+* list comprehensions
+* set comprehensions
+* dictionary comprehensions
+* await
+* yield, yield from
+* "..."
```

Comment:
what is that?

Reply:
Fancy slice notation I believe. I removed this bullet point since it is redundant.